tests/unit/common/buildApp.spec.ts

```diff
@@ -1,25 +1,47 @@
-import TermboxRequest from '@/common/TermboxRequest';
 import buildApp from '@/common/buildApp';
-import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
+import App from '@/components/App.vue';
+
+const mockGetChildComponents = jest.fn();
+jest.mock( '@/common/getChildComponents', () => ( {
+	__esModule: true,
+	default: ( app: any ) => mockGetChildComponents( app ),
+} ) );
+
+jest.mock( '@/store', () => ( {
```

Comment:
Assuming that `jest.resetAllMocks()` or the likes of which are called on a per-file basis.